### ISSUE TITLE
GameObjects/BG: GameObjects not visible

### ISSUE THREAD

**Issue by Killyana** [TrinityCore/TrinityCore#24033]
**Description:** Months ago I noticed some gobs are missing for battle grounds, like flags in Warsong Gulch

This happen randomly and generally when there's only on player in the BG, so it's probably from spawning gobs inside inactive grids.
But after some tests it turns out this gobs are just not visible: https://imgur.com/a/GxHOso4

**Steps to reproduce the problem:**

1. .deb BG
2. enter Warsong Gulch
3. wait until the bg start
4. check buffs and flags

**Branch(es):** 3.3.5

**TC rev. hash/commit:** rev. 7ffe01857110

**Comment by jackpoz** [comment]
any clue about when this issue started happening ?

**Comment by Killyana** [comment]
No, it's much older, just after dynamic spawns.

**Comment by Jildor** [comment]
yep, seems when you enter in wsg with only 1 player this happens, I try too with 2 players (1 alliance and 1 horde) and seems doesn't happens

**Comment by Faq** [comment]
So this is kinda only bg issue? One of reasons why this was started? #22700

**Comment by jackpoz** [comment]
Bg (Battleground) and Battlefields are 2 different things
